doc/Options/z390_filetypes.md

```diff
@@ -0,0 +1,32 @@
+ERR | MALE | ASCII | Output log of all START and ENDED messages plus any errors plus any error file references.|Text file with messages showing start and stop statistics for each step and any error messages issued by those steps.  A cross reference of all macro and copybook files with errors is included at the end.  For full cross reference of all macros and copybooks used see STA file.
+LOG | E | ASCII | Output log of for GUI or batch command execution controlled by CON option|Test file with messages issued udring program execution including start and stop statistics plus any errors and any WTO write to operator messages.
+LKD | L | ASCII | Optional input command for linker (INCLUDE, ENTRY, ALIAS, NAME)|Same format as BAL.
+MAC | M | ASCII | macro source files|Same format as BAL.
+MLC|| M | ASCII | macro assembler source programs read by mz390|Same format as BAL.
+MOD | LE | Binary | Binary code output from linker with no header and no RLD records.  This type file can be created using option MOD and can be loaded as a data table by Micro Focus CICS,|Binary file with all CSECT binary data.  For example DC C'ABC' using option MOD will result in 3 byte MOD file containing x'C1C2C3'.
+OBJ|AL|EBCDIC, ASCII|Output relocatible object code file for macro assembler with two format options. |The default is mainframe linker compatible relocatible object code 80 records (ESD, TXT, RLD, and END).  When OBJHEX option is specified for assembler, ASCII readable text extended format supporting 31 bit long sections is generated.  The z390 linker can read either format by examining file.
+OPT | MALE | ASCII | Optional options files referenced via @file option from command line or other option files|Text file with z390 options entered in free format delimited by spaces.  An * will terminate options on a line and start comments on that line. 
+PCH|M|ASCII|Output text files written via PUNCH statements in macro processor using optional ID=, DDNAME=, or DSNAME= extended parameters.  The default is pgm.PCH.|Text file output which is not limited to 80 byte BAL format using ALLOW.  Any macro program generating BAL output must format continuations.
+PRN|A|ASCII|Output assembly language listing from az390|Text file with the following listing fields:<br/>1) hex relative offset to instruction<br/>2) hex bytes in instruction or data with up ot 8 bytes per line.  Direction PRINT DATA will force listing all data 8 bytes per line.<br/>3) (file#/file-line#)generated-line# identified the file # and the relative file line #for the MLC, MAC, or CPY file that the generated line came from followed by the sequential generated line #.<br/>4) Flag field has + for macro generated lines, and = for line from copybook.<br/>5)BAL line with label, opcode, and operands.
```

Comment:
"<br/>5)BAL" -> "<br/"5) BAL". (add space between ) and B).

Reply:
Resolved by 52df1cf